mmv1/products/cloudrun/terraform.yaml

```diff
@@ -69,6 +69,15 @@ overrides: !ruby/object:Overrides::ResourceOverrides
       check_response_func_existence: PollCheckKnativeStatusFunc(res)
       actions: ['create', 'update']
     examples:
+      - !ruby/object:Provider::Terraform::Examples
+        name: "cloud_run_service_tasks"
+        primary_resource_id: "default"
+        vars:
+          project_id: "project-id"
```

Comment:
Can we move this into a `test_env_vars` block as is in the `cloud_run_service_basic` test? This will use the project that's set in the environment when running tests.

Reply:
That makes sense! Removing 👍🏼